### ISSUE TITLE
DesModder prevents Desmos iframes from loading

### ISSUE THREAD

**Issue by radian628** [DesModder/DesModder#287]
Whenever I have DesModder enabled, iframes that embed Desmos (from the Desmos site) do not load at all.

[This page reproduces this bug](https://htmlpreview.github.io/?https://gist.githubusercontent.com/radian628/2de91c1aadba833572e1029b65e7f4c0/raw/5bdfa080cd21a007700d230c22b9c048ed00f482/desmodder%2520iframe%2520bug)
````html
<!DOCTYPE html>
<html>
    <head>
    </head>
    <body>
        <header>
            <h1>Desmodder Iframe Bug</h1>
        </header>
        <iframe src="https://www.desmos.com/calculator/1c0qhuvra5"></iframe>
    </body>
</html>
````

**Comment by jared-hughes** [comment]
DesModder doesn't break non-iframe uses such as https://www.desmos.com/api/v1.7/docs/examples/screenshot.html because those use the public API, which is served from a different URL.

RuleCondition in https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/ appears to *not* allow filtering based on status as subframe/main frame, so the content script must be loaded in the subframe to re-load the calculator.js. Not sure why it isn't currently loaded; it's marked as `"matches": ["https://www.desmos.com/calculator*"]`, which doesn't have any special filtering for subframes.

**Comment by jared-hughes** [comment]
Fixed by #295.
